packages/experiments/src/components/SelectedItemsList/SelectedItemsList.tsx

```diff
@@ -1,131 +1,221 @@
 import * as React from 'react';
-import { Selection } from 'office-ui-fabric-react/lib/Selection';
+import { Selection, SELECTION_CHANGE } from 'office-ui-fabric-react/lib/Selection';
+
+import {
+  ISelectedItemsList,
+  IControlledSelectedItemsList,
+  IControlledSelectedItemListProps,
+  IUncontrolledSelectedItemListProps,
+  ISelectedItemsListProps,
+  BaseSelectedItem
+} from './SelectedItemsList.types';
+import { copyToClipboard } from '../../utilities/copyToClipboard';
+import { EventGroup } from '@uifabric/utilities/lib/EventGroup';
+
+/**
+ * Uses selection indeces as its own thing
+ * @param selection the selection to get the indeces of as state.
+ */
+const useSelectionIndeces = (selection: Selection): number[] => {
```

Comment:
The memoized Selection object + methods seems to suggest to me that Selections are mutated in place? In that case this may run into trouble as the selection indices won't update when the selection changes?

Reply:
Selections are mutated in-place, and issue resize events against EventGroups. Normally EventGroups are maintained by the fabric BaseComponent class, which initialize and dispose the group over a component's lifetime.